presto-docs/src/main/sphinx/clients/presto-console.rst

```diff
@@ -35,3 +35,23 @@ After starting Presto, you can access the web UI at the default port
 
 .. figure:: ../images/presto_console.png
    :align: center
+
+Plan Viewer
+=======================
+
+Plan Viewer is a tool to generate the graph of a query plan on the Presto Console.
+You can access it using this path: ``/ui/dev/index.html`` along with your server IP
+and port number. For example: ``http://localhost:8080/ui/dev/index.html``. When a
+Presto server restarts, all the query information is reset. However, the
+information of a query can be exported in JSON format using the ``JSON`` tab on
+the Query Details page and saved locally. Here is where the ``JSON`` tab is located.
+
+.. figure:: ../images/query_details_json.png
+   :align: center
+
+With the JSON files, you can use the Plan Viewer to view the graph of query plans.
+
+1. Access the Plan Viewer page. For example: ``http://localhost:8080/ui/dev/index.html``
+2. Click on the ``Browse`` button and select the JSON file you want to examine.
```

Comment:
```suggestion
2. Select the ``Browse`` button and select the JSON file you want to examine. The query plan graph is displayed. 
```
See the [Gitlab documentation style guide recommended word list entry for click](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#click) for discussion and suggestions. 

Moved "The query plan graph is displayed." to here, as it's the result or output of the user's action in step 2.

Reply:
lesson learned, thanks for the info